src/app/components/header/header.component.html

```diff
@@ -13,57 +13,52 @@ <h5 class="offcanvas-title" id="offcanvasNavbarLabel">Menu</h5>
       </div>
       <div class="offcanvas-body">
         <ul class="navbar-nav justify-content-start flex-grow-1 pe-3">
-          <li class="nav-item">
-            <a class="nav-link active" aria-current="page" routerLink="home" data-bs-dismiss="offcanvas">Home</a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" routerLink="weather" data-bs-dismiss="offcanvas">Alerts</a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" routerLink="help" data-bs-dismiss="offcanvas">Help</a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="#">Map</a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="#">Prior Tickets</a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="#">Tools</a>
-          </li> <hr>
-          <li class="nav-item">
-            <a class="nav-link" href="#">Timesheet</a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="#">System Health <img class="float-end" src="assets/idkweather.png">
-            </a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="#">News</a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="#">Safety</a>
-          </li>
-          <li class="nav-item">
-            <a class="nav-link" href="#">Profile</a>
-          </li>
-          <li class="nav-item dropdown">
-            <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
-              Mini-projects
-            </a>
-            <ul class="dropdown-menu">
-              <li><a class="dropdown-item" routerLink="we-track">weTrack</a></li>
-              <li>
-                <hr class="dropdown-divider">
+          <!-- this should be its own components, it works but very messy -->
```

Comment:
That's what i said!

Reply:
I thought you said "that's what she said"